.php_cs

```diff
@@ -21,7 +34,10 @@ $config = Config::create()
     ->setUsingLinter(false);
 
 $finder = $config->getFinder()
-    ->in('src')
-    ->in('test');
+    ->in(__DIR__)
```

Comment:
This fails if you've ever done a `bin/build` because there's a second vendor directory just for phar builds called `build-vendor`, and it checks that. Mind adding that as an exclusion?

(And yeah, I really should test phar builds on CI)